specs/phase1/shard-transition.md

```diff
@@ -50,6 +50,9 @@ def verify_shard_block_message(beacon_state: BeaconState,
                                shard: Shard) -> bool:
     assert block.shard_parent_root == shard_state.latest_block_root
     assert block.slot == slot
+    next_slot = Slot(beacon_state.slot + 1)
+    offset_slots = compute_offset_slots(get_latest_slot_for_shard(beacon_state, shard), next_slot)
+    assert slot in offset_slots
```

Comment:
Another thought, should we assert `block.beacon_parent_root` here? (using `beacon_state`)